### PR TITLE
DEL: Remove isCatalyst package

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/z2u3t3mvvf5eutycfk3d39ajh8?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/catal-1622268692.apk)

**Review by limpbrains — APPROVED** [review]

File: blue_modules/environment.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import { getSystemName, isTablet } from 'react-native-device-info';
-import isCatalyst from 'react-native-is-catalyst';
+import { getSystemName, isTablet, getDeviceType } from 'react-native-device-info';
 
 const isMacCatalina = getSystemName() === 'Mac OS X';
 
 module.exports.isMacCatalina = isMacCatalina;
-module.exports.isCatalyst = isCatalyst;
+module.exports.isDesktop = getDeviceType() === 'Desktop';
```
Comment (Overtorment) [inline blue_modules/environment.js:6]: just a thought. is it worth caching value once instead of calculating it every time in runtime?

Comment (Overtorment) [reply blue_modules/environment.js:6]: like, how heavy is that `getDeviceType` call?

File: package.json [hunk 2]
```diff
@@ -139,7 +139,6 @@
     "react-native-idle-timer": "https://github.com/BlueWallet/react-native-idle-timer#8587876d68ab5920e79619726aeca9e672beaf2b",
     "react-native-image-picker": "3.5.0",
     "react-native-inappbrowser-reborn": "https://github.com/BlueWallet/react-native-inappbrowser#fa2d8e1763e46dd12a7e53081e97a0f908049103",
-    "react-native-is-catalyst": "https://github.com/BlueWallet/react-native-is-catalyst#v1.0.0",
```
Comment (Overtorment) [inline package.json:142]: btw need to not forget delete old repos on our org after we stop using them

**Review by Overtorment — APPROVED** [review]
lgtm, but conflicts

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/yw1ebf8vy1p3kyg3pffvdzn8fm?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/catal-1622314583.apk)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**